org.eclipse.acute/src/org/eclipse/acute/AcutePreferenceInitializer.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *  Lucas Bullen   (Red Hat Inc.) - Initial implementation
+ *******************************************************************************/
+package org.eclipse.acute;
+
+import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer;
+import org.eclipse.jface.preference.IPreferenceStore;
+
+public class AcutePreferenceInitializer extends AbstractPreferenceInitializer {
+
+	public static String useExplicitDotnetPathPreference = "dotnet.useExplicitPath";
+	public static String explicitDotnetPathPreference = "dotnet.explicitPath";
+
+	@Override
+	public void initializeDefaultPreferences() {
+		IPreferenceStore store = AcutePlugin.getDefault().getPreferenceStore();
+		store.setDefault(useExplicitDotnetPathPreference, false);
+		store.setDefault(explicitDotnetPathPreference, "");
```

Comment:
Here should be some logic to set a decent default value, such as `dotnet` on Linux or more advanced strategies.